src/utils/cloud_storage.R

```diff
@@ -198,3 +201,9 @@ blob_prod <- az$blob_container(
   endpoint = blob_endpoint_prod,
   name = "hdx-signals-mc"
 )
+
+# blob object for WFP, where the market monitoring data is stored
+blob_wfp <- az$blob_container(
+  endpoint = blob_endpoint_dev,
```

Comment:
Is there a reason we wouldn't put this in the prod blob container? It'd be great to keep things centralized there.

Reply:
Yes, @isatotun can hopefully get this into prod but previously stuff she was working on was put into `dev` due to access issues she was having that needed solutions from Saeid. Will create a separate issue because we want to shift everything she's using from `dev` to `prod`.

Reply:
Copy that 🫡